File: Volume 11 (1100 - 1199)/1131/1131.rb
#!/usr/bin/env ruby

s = 1
gremio = inter = empates = total = 0
while s == 1
    a, b = gets.split(' ').map(&:to_i)
    if a > b
        inter += 1
    elsif b > a
        gremio += 1
    else
        empates += 1
    end
    total += 1
    puts "Novo grenal (1-sim 2-nao)"
    s = gets.to_i
end
puts "%d grenais" % [total]
puts "Inter:%d" % [inter]
puts "Gremio:%d" % [gremio]
puts "Empates:%d" % [empates]
if inter > gremio
    puts "Inter venceu mais"
elsif gremio > inter
    puts "Gremio venceu mais"
else
    puts "Nao houve vencedor"
end
gets

File: Volume 11 (1100 - 1199)/1131/1131.c
#include <stdio.h>
 
int main() {
    int a, b, c = 1, gremio = 0, inter = 0, empates = 0, total = 0;
    while (c == 1) {
        scanf("%d %d", &a, &b);
        if (a > b) inter++;
        else if (b > a) gremio++;
        else empates++;
        total++;
        printf("Novo grenal (1-sim 2-nao)\n");
        scanf("%d", &c);
    }
    printf("%d grenais\n", total);
    printf("Inter:%d\n", inter);
    printf("Gremio:%d\n", gremio);
    printf("Empates:%d\n", empates);
    if (inter > gremio) printf("Inter venceu mais\n");
    else if (gremio > inter) printf("Gremio venceu mais\n");
    else printf("Nao houve vencedor\n");
    return 0;
}